system/src/system_network_manager.cpp

```diff
@@ -812,6 +830,25 @@ void NetworkManager::resetInterfaceProtocolState(if_t iface) {
     }
 }
 
+int NetworkManager::waitModemOff(if_t iface, system_tick_t timeout) const {
```

Comment:
Let's generalize this. This is waiting for interface to turn off, not modem specifically.